src/test/rpc/LedgerRPC_test.cpp

```diff
@@ -1222,7 +1222,7 @@ class LedgerRPC_test : public beast::unit_test::suite
     void
     testLedgerEntryUnknownOption()
```

Comment:
Please:

- Add a parameter for `api_version`.
- Call this method in a loop over versions from `apiMinimumSupportedVersion` to `apiBetaVersion`.
- Remove the other method (`testLedgerEntryInvalidParams`).
- Make the assertion on the error value conditional on the `api_version` parameter, using condition `< 2` for the old behavior and `else` for the new behavior.

Reply:
Thanks @thejohnfreeman!